packages/ui/core-components/src/lib/organisms/layout/BreadCrumbs.svelte

```diff
@@ -58,7 +58,7 @@
 </script>
 
 <div class="flex items-start mt-0 whitespace-nowrap overflow-auto">
-	<div class="inline-flex items-center text-sm capitalize gap-1 text-gray-500 mb-2 sm:mb-4">
+	<div class="inline-flex items-center text-sm capitalize gap-1 text-base-content/50 mb-2 sm:mb-4">
```

Comment:
`text-base-content/50` is a smell to me we've got this ~35 times, shouldn't we have something semantic, e.g. muted-content, muted-background?

Reply:
Agree 100%